### PR TITLE
Add security policy

### PR DESCRIPTION
**Pull request by johnrballard**
This adds a security policy to the repository to report security issues via the github security vulnerability feature.

### CODE REVIEW

File: security.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Security
+
+## Reporting Security Issues
+
+**Please do not report security vulnerabilities through public GitHub issues.**
+
+Instead, please open a [security advisory][advisory] which notify the maintainers. You should receive a response within 3 working days. If for some reason you do not, please follow up via email to ensure we received your original message.
```
Comment (workingjubilee) [inline security.md:7]:
```suggestion
Instead, please open a [security advisory][advisory] to notify the maintainers. You should receive a response within 3 working days. If for some reason you do not, please follow up via email to ensure we received your original message.
```

**Comment by eeeebbbbrrrr** [comment]
PR contains changes to README.md that it shouldn't.  @johnrballard I think this PR needs to be rebased.  Would you mind doing that?  You'll have to force-push the branch.

**Comment by johnrballard** [comment]
> PR contains changes to README.md that it shouldn't. @johnrballard I think this PR needs to be rebased. Would you mind doing that? You'll have to force-push the branch.

@eeeebbbbrrrr I have updated the source branch to point to develop.

**Review by eeeebbbbrrrr — APPROVED** [review]

**Comment by eeeebbbbrrrr** [comment]
Thanks, @johnrballard.  This was cherry-picked into `master` so it's live on the repo landing page too.

**Comment by eeeebbbbrrrr** [comment]
Note, I renamed this to `SECURITY.md` in a separate commit.  I would have just asked for that in the original PR but I didn't notice.

https://github.com/pgcentralfoundation/pgrx/blob/master/SECURITY.md

**Pull request merged**